src/containers/NahmiiHoc/combined-selectors.js

```diff
@@ -65,6 +65,17 @@ const requiredDataLoading = (supportedAssets, prices, walletBalances) => (
   || walletBalances.get('loading')
 );
 
+const findPriceInfo = (prices, currency) => {
+  const priceInfo = prices
+    .get('assets')
+    .find((p) => p.get('currency') === currency);
```

Comment:
Is `currency` normalised when it comes to casing? Otherwise this may not be able to compare currencies correctly.

Reply:
Good catch, thanks. I pushed an update to compare it using case-insensitive.